Makefile

```
@@ -214,7 +214,7 @@ make-migrations:
 	$(DJANGO_MANAGE) makemigrations api reporting reporting_common cost_models
 
 delete-db:
-	$(PREFIX) rm -rf $(TOPDIR)/pg_data/*
+	$(PREFIX) rm -rf $(TOPDIR)/pg_data/data
```

Just delete the contents of the directory.
```suggestion
	$(PREFIX) rm -rf $(TOPDIR)/pg_data/data/*
```